DependencyInjection/FOSUserExtension.php

```diff
@@ -207,4 +211,12 @@ protected function remapParametersNamespaces(array $config, ContainerBuilder $co
             }
         }
     }
+
+    protected function setForceLoginOnPasswordChange(ContainerBuilder $container)
+    {
+        foreach ($container->findTaggedServiceIds('fos_user.user_provider') as $id => $tags) {
```

Comment:
Seemed to be cleaner to do as a tag instead of hard coding to 2 difference services.